internal/database/models/valuesetconcept.go

```diff
@@ -0,0 +1,3 @@
+package models
+
+// no custom methods; see xo/valuesetconcept.xo.go
```

Comment:
we can delete all these empty files; left for now until we have something in a ReadMe spelling out the distinction between custom model methods and the xo-generated ones